webapp/src/routes/SpendTools/styles.js

```diff
@@ -0,0 +1,205 @@
+export default theme => ({
+  root: {
+    margin: '8px 16px'
+  },
+  toolInformation: {
+    marginTop: '24px'
+  },
+  formContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'center',
+    alignItems: 'flex-start',
+    gap: '16px',
+    padding: '8px 16px',
+    margin: '16px 0 8px 0',
+    [theme.breakpoints.down('sm')]: {
+      flexDirection: 'column',
+      alignItems: 'center',
+      gap: 0
+    }
+  },
+  inputContainer: {
+    height: '74px',
+    width: '100%',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'flex-end',
+    alignItems: 'flex-start',
+    gap: '4px',
+    '& .MuiFormLabel-root': {
+      fontSize: '14px',
+      lineHeight: '1.57',
+      letterSpacing: '-0.28px',
+      color: '#667080'
+    },
+    '& .MuiInputBase-input': {
+      padding: '13px 25px 13px 25px'
+    },
+    '& .MuiOutlinedInput-notchedOutline': {
+      border: 'solid 1px #667080'
+    },
+    '& small': {
+      color: 'red'
+    }
+  },
+  specialInput: {
+    width: '100%',
+    '& #labels-id': {
+      display: 'flex',
+      justifyContent: 'space-between'
+    },
+    '& #amount-id': {
+      fontSize: '14px',
+      lineHeight: '1.57',
+      letterSpacing: '-0.28px',
+      color: '#667080'
+    },
+    '& #available-id': {
+      fontSize: '12px',
+      lineHeight: 1.83,
+      letterSpacing: '-0.24px'
+    },
+    '& .MuiInputBase-input': {
+      padding: '13px 25px 13px 25px'
+    },
+    '& .MuiOutlinedInput-notchedOutline': {
+      border: 'solid 1px #667080'
+    },
+    '& small': {
+      color: 'red'
+    },
+    '& .css-c8obxa-MuiFormHelperText-root': {
+      color: '#000'
+    }
+  },
+  rowFormContainer: {
+    height: '202px',
+    width: '100%',
+    flexGrow: 0,
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'flex-start',
+    alignItems: 'flex-start',
+    gap: '16px',
+    padding: '8px 16px',
+    [theme.breakpoints.up('xs')]: {
+      maxWidth: '377px'
+    }
+  },
+  selectForm: {
+    alignSelf: 'stretch',
+    height: '48px',
+    borderRadius: '6px',
+    '& .css-1d3z3hw-MuiOutlinedInput-notchedOutline': {
```

Comment:
I'm pretty sure if You use the correct MUI class you don't need to write that long css classes

example: 
`.MuiOutlinedInput-notchedOutline`